frontend/user/vue-components/ApplicationLabel.vue

```diff
@@ -1,42 +1,44 @@
 <template>
   <ul v-if="currentApp !== null" class="nav navbar-nav">
-    <li class="dropdown">
+    <li class="dropdown notifications-menu">
```

Comment:
removing the menu "section title" means that you don't need notification menu anymore.